crm_lead_code/__openerp__.py

```diff
@@ -1,48 +1,28 @@
-# -*- encoding: utf-8 -*-
-##############################################################################
-#
-# Copyright (c)
-#    2015 Serv. Tec. Avanzados - Pedro M. Baeza (http://www.serviciosbaeza.com)
-#    2015 AvanzOsc (http://www.avanzosc.es)
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as published
-#    by the Free Software Foundation, either version 3 of the License, or
-#    (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# -*- coding: utf-8 -*-
+# © 2015 Tecnativa - Pedro M. Baeza (https://www.tecnativa.com)
+# © 2015 AvanzOsc (http://www.avanzosc.es)
+# © 2016 Atul Arvind (http://tech.heliconia.in)
+
 
 {
     "name": "Sequential Code for Leads / Opportunities",
-    "version": "8.0.1.0.0",
+    "version": "9.0.1.0.0",
     "category": "Customer Relationship Management",
     "author": "OdooMRP team, "
               "AvanzOSC, "
-              "Serv. Tecnol. Avanzados - Pedro M. Baeza, "
+              "Tecnativa - Pedro M. Baeza, "
+              "Atul Arvind, "
```

Comment:
Please remove from original authors. IMO shouldn't also be added to copyright owners on file disclaimers.